billing.md

```diff
@@ -99,34 +93,119 @@ Next, add the `Billable` trait to your model definition. This trait provides var
         use Billable;
     }
 
+Cashier assumes your Billable model will be the default `App\User` class that ships with Laravel. If you wish to change this you can update this in your `.env` file:
+
+    CASHIER_MODEL="App\User"
```

Comment:
the quotes of `"App\User"` cause the app to show the death white screen!

Reply:
Does `CASHIER_MODEL=App\User` work instead?

Reply:
Yes that worked.